src/test/java/bio/terra/janitor/app/controller/JanitorApiControllerTest.java

```diff
@@ -341,6 +341,121 @@ public void abandonThenBumpResources() throws Exception {
     assertEquals(TrackedResourceState.READY.name(), newerResource.getState());
   }
 
+  @Test
+  public void bumpResources() throws Exception {
```

Comment:
I think we have been and and are getting a little worse on what should be a `JanitorServiceTest` vs a `JanitorApiControllerTest`.  Business logic tests like "what does bump do" should be `JanitorServiceTest`. The only thing we need to test as a `JanitorApiControllerTest` is serialization - is the request/response format as expected? Arguably, those are often not very valuable tests once we have a couple examples - it's mostly handled by our libraries; if one serialization works, they probably all work (as long as they're "the same").

So, I think this test is misplaced in that it's doing too much about how create/abandon/bump interact. OTOH, we're maybe doing a little too much of that already in this file, and we do need to set up a little bit in order for "bump" to succeed at all. I'd advocate for making this a test in `JanitorServiceTest` instead, but let me know if you disagree.

Reply:
Yeah, I also realized that those tests should be done in `JanitorServiceTest`, I thought getting a chance to check with you then refactor those tests. But for now in this PR, I can move abandAndBump to  `JanitorServiceTest`, PTAL, thanks.